packages/cloudflare/src/cli/build/patches/ast/optional-deps.ts

```diff
@@ -16,7 +16,7 @@ rule:
     pattern: $MOD
     kind: string_fragment
     stopBy: end
-    regex: ^caniuse-lite(/|$)
+    regex: ^caniuse-lite(/|$)|jimp(/|$)|probe-image-size(/|$)
```

Comment:
are you sure about this regex? the `^` only applies to `caniuse-lite` so the regex also matches things just ending with for example `jimp` (e.g. like `hello-jimp`)

maybe this would be more appropriate?
```suggestion
    regex: ^(caniuse-lite|jimp|probe-image-size)(/|$)
```

Reply:
good catch!